src/index.html

```diff
@@ -1,12 +1,388 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="MoyoLogo"
+          class="header__img"
+        >
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="index.html" class="header__item-link is-active"> 
```

Comment:
```suggestion
            <a href="index.html" class="header__item-link header__item-link--active"> 
```

Reply:
i can't delete class is-active, because it's responsible for references and tests.
it's written in checklist 
<img width="491" alt="Знімок екрана 2023-02-06 о 23 12 56" src="https://user-images.githubusercontent.com/81996837/217089289-f7dee135-3def-4565-aef3-430b3d2e4c8f.png">

Reply:
Ok, let it be as is.